test/imminence/imminence_api_pact_test.rb

```diff
@@ -11,45 +11,47 @@
 
     it "responds with all responses for the given dataset" do
       imminence_api
+        .given("a service exists called number-plate-supplier with places")
         .upon_receiving("the request to retrieve all places for the current dataset")
         .with(
           method: :get,
           path: "/places/number-plate-supplier.json",
           headers: GdsApi::JsonClient.default_request_headers,
+          query: { lat: "1.11", lng: "2.22", limit: "5" },
```

Comment:
Could fee44565e66f83c7854a254108d25265ade70f5e be rebased into cdc3135a75555a948c918115f26f90a1c99fafaa?

Reply:
Sorted 👍